Setup/KuduSetupCustomWebPiFeed.xml

```diff
@@ -212,4 +208,123 @@
       </installer>
     </installers>
   </entry>
-</feed>
\ No newline at end of file
+  <!-- entry for iisnode locked to a known version -->
+  <entry>
+    <productId>iisnode_0_2_11</productId>

```

Comment:
correct. it has the old version from 2012, and the old version is depends on Node.js 0.8